ls1intum/Artemis
# `Programming exercises`: Update Ares to 1.10.1

### Checklist
#### General


#### Changes affecting Programming Exercises

### Motivation and Context
A new version of Ares has been released:
- https://github.com/ls1intum/Ares/releases/tag/1.10.0
- https://github.com/ls1intum/Ares/releases/tag/1.10.1

This new release can be used in Artemis once https://github.com/ls1intum/artemis-maven-docker/pull/87 has been merged and a new release of the Docker image the tests run in has been created.

### Description
Updates Ares as used in the Java and Kotlin templates to the new version.

### Steps for Testing
Prerequisites:
- 1 Instructor

1. Create a new Java programming exercise with the Maven project template.
2. The solution repo should still pass all tests.
3. Repeat the steps with the Gradle project template.
4. Repeat the steps with a Kotlin exercise.

### Review Progress





#### Manual Tests
- [x] Test 1 on a test server connected to Bamboo
- [x] Test 2 on a test server connected to Jenkins
